docs/collaborations.md

```diff
@@ -115,6 +120,7 @@ Get Pending Collaborations
 A collection of all the user's pending collaborations can be retrieved with
 `CollaborationsManager.GetPendingCollaborationAsync(IEnumerable<string> fields = null)`.
 
+<!-- sample get_collaborations -->
```

Comment:
This will conflict with the method above; both hit the same endpoint, just with different query params.  Should this be a variant, e.g. `get_collaborations pending`?

Reply:
Not really. Turns out the other one was `get_folders_id_collaborations`, and this one doesn't need to be a variation cause I think `get_collaborations` only ever returns pending ones.